services/QuillLMS/app/workers/geocoder_timezone_worker.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class GeocoderTimezoneWorker
+  include Sidekiq::Worker
+
+  def perform(user_id, ip_address)
+    user = User.find_by_id(user_id)
+    return unless user && ip_address.present?
```

Comment:
Technically we should be persisting `ip_address` on the User model, so should be able to fetch it if it wasn't passed, but I think that passing its value at time of enqueue seems reasonable for weird edge cases where it might be updated while the job is on the queue.  I think that in such cases we want to use the value at time of enqueue and not whatever is "current"?

Reply:
👀